### PR TITLE
Improve CLI for log tracer

### PR DESCRIPTION
**Pull request by terry-xiaoyu**
### CLI
```shell
--------------------------------------------------------------------------------
trace list                                      # List all traces started
trace start client <ClientId> <File> [<Level>]  # Trace logs related to a client_id
trace stop client  <ClientId>                   # Stop tracing a client by client_id
trace start topic  <Topic>    <File> [<Level>]  # Trace logs related to a topic
trace stop topic   <Topic>                      # Stop tracing for a topic
--------------------------------------------------------------------------------
```

### Trace Examples

```shell
## change the primary log level to all
## this is important before tracing logs
$ emqx_ctl log primary-level all
ok

## start tracing logs related to clientid 'client1', and redirect the messages
## to '/tmp/client1.log'
$ emqx_ctl trace start client 'client1' '/tmp/client1.log'
ok

## same to above, expect that only trace messages higher than error
$ emqx_ctl trace start client 'client1' '/tmp/client1.log' error
ok

## start tracing logs related to topic filter 'topic1/#', and redirect the messages
## to '/tmp/topic1.log'. wildcard is supported.
$ emqx_ctl trace start topic 'topic1/#' '/tmp/topic1.log'
ok
```

### CODE REVIEW

**Comment by Gilbert-Wong** [comment]
Please align the example of command usage.

File: src/emqx_mgmt_cli.erl [hunk 1]
```diff
@@ -535,31 +535,37 @@ mnesia(_) ->
...
-    emqx_cli:usage([{"trace list",                       "List all traces"},
-                    {"trace client <ClientId> <LogFile>","Trace a client by client_id"},
-                    {"trace client <ClientId> off",      "Stop tracing a client by client_id"},
-                    {"trace topic <Topic> <LogFile>",    "Trace a topic"},
-                    {"trace topic <Topic> off",          "Stop tracing a Topic"}]).
-
-trace_on(Who, Name, LogFile) ->
-    case emqx_tracer:start_trace({Who, iolist_to_binary(Name)}, LogFile) of
+    emqx_cli:usage([{"trace list", "List all traces started"},
+                    {"trace start client <ClientId> [<Level>] <File>", "Trace logs related to a client_id"},
```
Comment (spring2maz) [inline src/emqx_mgmt_cli.erl]: I would reword to:
Traces for a client
Traces for a topic

Having the optional arg in the middle is a bit counterintuitive.
We will be in a very awkward corner when it’s time to add another optional arg

Comment (terry-xiaoyu) [reply src/emqx_mgmt_cli.erl]: fixed
